README.md

```diff
@@ -3,24 +3,30 @@ barebones setup for JavaScript / Node projects replace this with a one-liner phr
 [![Codacy Badge](https://img.shields.io/badge/Code%20Quality-D-red)](https://img.shields.io/badge/Code%20Quality-D-red)
 
 
-## About
-
-What is this project about. Ok to enrich here or the section above it with an image. 
-
-Once this repo has been setup on Codacy by the TTL, replace the above badge with the actual one from the Codacy dashboard, and add the code coverage badge as well. This is mandatory
-
-This is a simple JavaScript starter repo template for setting up your project. The setup contains
-
-- Jest: For runnung tests. We strongly recommend that JavaScript projects use Jest
-
-- Eslint & Prettier: For formatting code to match ESlint AirBnB coding standard. You might need to install the ESlint and Prettier plugins for your code editor to make the most of these utilities
-
-## Why
-
-Talk about what problem this solves, what SDG(s) and SGD targets it addresses and why these are imoirtant
-
-## Usage
- How would someone use what you have built, include URLs to the deployed app, service e.t.c when you have it setup
```

Comment:
The usage should state how someone can use this app. It can include the steps to clone the repo and start the dev server. e.g.
Get started by cloning the repo 'git clone <url>'
change into the directory: cd <foldername>
Start dev server: yarn dev or npm run dev
Start liniting: yarn lint or npm run lint.

or you can just give the direction to the hosted app provided by the frontend guys.